Library/Homebrew/requirements/xorg_requirement.rb

```diff
@@ -0,0 +1,24 @@
+require "requirement"
+
+class XorgRequirement < Requirement
+  fatal true
+  default_formula "linuxbrew/xorg/xorg"
+
+  env { ENV.x11 }
+
+  def initialize(name = "xorg", tags = [])
+    @name = name
+    if /(\d\.)+\d/ === tags.first
+      tags.shift
```

Comment:
this means that we chop out the version number for compatibility with X11Requirement, but we don't store it since the XQuartz version numbers are meaningless on Linux.